types/ramda/index.d.ts

```diff
@@ -1000,6 +1000,7 @@ declare namespace R {
          * additional call to fn; instead, the cached result for that set of arguments will be returned.
          */
         memoize(fn: (...a: any[]) => any): (...a: any[]) => any;
+        memoize<T>(fn: (...a: any[]) => T): (...a: any[]) => T;
```

Comment:
This overload does not have associated documentation. Consider copying the documentation from the preceding overload.